### PR TITLE
Laboratory work #1, Veronika Tsytsulina - 23FPL1

### PR DESCRIPTION
**Pull request by VeronicaTs30**

### CODE REVIEW

File: lab_1_classify_by_unigrams/main.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 
 def tokenize(text: str) -> list[str] | None:
+
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/main.py:10]: revert

File: lab_1_classify_by_unigrams/main.py [hunk 2]
```diff
@@ -34,7 +42,13 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
-
+    if not isinstance(tokens, list) or not all(isinstance(token, str) for token in tokens):
+        return None
+    freq_dictionary = {}
+    total_amount_of_tokens = len(tokens)
+    for elem in set(tokens):
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/main.py:49]: Perfect

File: lab_1_classify_by_unigrams/start.py [hunk 3]
```diff
@@ -3,20 +3,25 @@
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
 
-
+from main import tokenize
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/start.py:6]: add an empty line

File: lab_1_classify_by_unigrams/start.py [hunk 4]
```diff
@@ -3,20 +3,25 @@
...
     with open("assets/texts/unknown.txt", "r", encoding="utf-8") as file_to_read_unk:
         unknown_text = file_to_read_unk.read()
-    result = None
+        text_in_tokens = tokenize(unknown_text)
+    result = text_in_tokens
+    print(result)
     assert result, "Detection result is None"
 
 
+
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/start.py:25]: revert

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
What about 6?

File: lab_1_classify_by_unigrams/start.py (hunk 3 above)
Comment (VeronicaTs30) [reply lab_1_classify_by_unigrams/start.py]: done

File: lab_1_classify_by_unigrams/main.py (hunk 1 above)
Comment (VeronicaTs30) [reply lab_1_classify_by_unigrams/main.py]: done

File: lab_1_classify_by_unigrams/main.py [hunk 5]
```diff
@@ -4,9 +4,11 @@
 Language detection
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
-
+a = 13
```
[the remark this answers is not in the document]
Comment (VeronicaTs30) [reply lab_1_classify_by_unigrams/main.py]: done

File: lab_1_classify_by_unigrams/main.py (hunk 2 above)
Comment (VeronicaTs30) [reply lab_1_classify_by_unigrams/main.py:48]: done

**Review by artyomtugaryov — DISMISSED** [review]
What about 8?

**Pull request closed without merging**